src/liapunovspectrum.jl

```diff
@@ -98,26 +98,28 @@ function liap_jetcoeffs!(eqsdiff!, t::Taylor1{T}, x::Vector{Taylor1{U}},
         dx::Vector{Taylor1{U}}, xaux::Vector{Taylor1{U}},
         δx::Array{TaylorN{Taylor1{U}},1}, dδx::Array{TaylorN{Taylor1{U}},1},
         jac::Array{Taylor1{U},2}) where {T<:Real, U<:Number}
-
     order = x[1].order
-
     # Dimensions of phase-space: dof
     nx = length(x)
     dof = round(Int, (-1+sqrt(1+4*nx))/2)
-
     for ord in 0:order-1
         ordnext = ord+1
 
         # Set `taux`, auxiliary Taylor1 variable to order `ord`
-        taux = Taylor1(t.coeffs[1:ordnext])
+        @inbounds taux = Taylor1( t.coeffs[1:ordnext] )
         # Set `xaux`, auxiliary vector of Taylor1 to order `ord`
         for j in eachindex(x)
             @inbounds xaux[j] = Taylor1( x[j].coeffs[1:ordnext] )
         end
 
         # Equations of motion
-        eqsdiff!(taux, xaux, dx)
-        stabilitymatrix!( eqsdiff!, t[0], view(xaux,1:dof), δx, dδx, jac )
+        for ind in eachindex(δx)
+            @inbounds δx[ind] = xaux[ind] + _δv[ind]
```

Comment:
What is `_δv`? I guess it is some global variable; if so, maybe it is better to pass it as an argument in the function call.

This for loop may be merged with the one above (`x` and `δx` have the same length).

Reply:
Exactly, `_δv` is the global variable declared in `liap_taylorinteg`; I also thought of merging the loops but actually `δx` has the length of the vector of initial conditions (say, `q0`) whereas `x` has the length of `q0` plus the size of the stability matrix

Reply:
I'll try to pass `_δv` as an argument and see if performance improves

Reply:
(I actually noticed this but still added the comment)

Ok, with the two separated loops.